VOID PACKAGING NOTE: This was added to our release at upstream's request.
Upstream released, found a bug, fixed it, and reached out to their downstream
packagers to get this fix included.

This is instead of breaking downstream packagers by moving the tag. We thank
neomutt upstream for their attention to packaging this software.

commit 9e7537caddb9c6adc720bb3322a7512cf51ab025 (HEAD -> master, origin/master, origin/HEAD)
Author: Richard Russon <rich@flatcap.org>
Date:   Fri Apr 17 15:36:58 2020 +0100

    ensure OP_NULL is always first

    A recent refactoring altered the ordering of the OPs when autocrypt was
    enabled.  This mean that OP_NULL wasn't 0 any more.

    Fixes: #2268

diff --git a/opcodes.h b/opcodes.h
index 701df71f8..3984d3a03 100644
--- neomutt-20200417/opcodes.h
+++ neomutt-20200417/opcodes.h
@@ -37,7 +37,6 @@
 #endif

 #define OPS_CORE(_fmt) \
-  _fmt(OP_NULL,                           N_("null operation")) \
   _fmt(OP_ATTACH_COLLAPSE,                N_("toggle display of subparts")) \
   _fmt(OP_ATTACH_VIEW_MAILCAP,            N_("force viewing of attachment using mailcap")) \
   _fmt(OP_ATTACH_VIEW_TEXT,               N_("view attachment as text")) \
@@ -317,6 +316,7 @@
   _fmt(OP_COMPOSE_SMIME_MENU,             N_("show S/MIME options")) \

 #define OPS(_fmt) \
+  _fmt(OP_NULL,                           N_("null operation")) \
   OPS_AUTOCRYPT(_fmt) \
   OPS_CORE(_fmt) \
   OPS_SIDEBAR(_fmt) \

